tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -1055,21 +1055,7 @@ public static CLDRLocale validateLocale(PrintWriter out, String loc, String sess
          * loc can have either - or _; normalize to _
          */
         loc = loc.replace('-', '_');
```

Comment:
do we need this line ? or maybe it should also go into a utility function?

Reply:
I think we have to assume it's needed... ideally it should be encapsulated in a module specific to locales, not SurveyAjax; and as much as possible we should pass around CLDRLocale objects instead of strings...